src/newViews/Tagging/mockData.ts

```diff
@@ -0,0 +1,154 @@
+import { Entry } from './types';
+
+const values: Entry[] = [
+    {
+        id: 1,
+        createdAt: 'Fri Feb 26 2021 20:14:55 GMT+0545 (Nepal Time)',
+        createdBy: 65277,
+        createdByName: 'sameer',
+        modifiedBy: 34089,
+        modifiedByName: 'sameer',
+        clientId: 'wwfrillnhe',
+        versionId: 74123,
+        project: 9839,
+        lead: 64415,
+        analyticalFramework: 96021,
+        attributes: [
+            {
+                id: 'widget1',
+                type: 'text',
+                data: {
+                    value: 'ram',
+                },
+            },
+            {
+                id: 'widget2',
+                type: 'number',
+                data: {
+                    value: 10,
+                },
+            },
+            {
+                id: 'widget4',
+                type: 'single-select',
+                data: {
+                    value: 'apple',
+                },
+            },
+            {
+                id: 'widget5',
+                type: 'multi-select',
+                data: {
+                    value: ['apple', 'ball'],
+                },
+            },
+            {
+                id: 'widget6',
+                type: 'date',
+                data: {
+                    value: 'Fri Jul 30 2021 15:06:48 GMT+0545 (Nepal Time)',
+                },
+            },
+            {
+                id: 'widget7',
+                type: 'time',
+                data: {
+                    value: '1627636952267',
+                },
+            },
+            {
+                id: 'widget8',
+                type: 'date-range',
+                data: {
+                    value: {
+                        startDate: 'Fri Jul 30 2021 15:06:48 GMT+0545 (Nepal Time)',
+                        endDate: 'Sat Jul 31 2021 15:09:32 GMT+0545 (Nepal Time)',
+                    },
+                },
+            },
+            {
+                id: 'widget9',
+                type: 'time-range',
+                data: {
+                    value: {
+                        startTime: '1627636952267',
+                        endTime: '1627637267565',
```

Comment:
I think these should be number. Let's check with server as well.

Reply:
@tnagorra I have written them as a string to conform with AF typings

Reply:
fixed